### ISSUE TITLE
EVcomplex database file downloading / generation

### ISSUE THREAD

**Issue by thomashopf** [debbiemarkslab/EVcouplings#96]
Necessary for EVcomplex part of pipeline to be usable.

Two escalation levels:

1) Distribute pre-generated downloadable file using DB update tool (download from lab website)

2) Include script that generates databases from source files in fully automated way.

For next release, I think having 1) would be sufficient.

**Comment by thomashopf** [comment]
Just a bump on this, once this is ready (escalation level 1) we are ready for a new release

**Comment by aggreen** [comment]
Pre-generated files made available by #124. We still need to make scripts to automate download, but this is currently lower priority.
